ubireader/ubifs/list.py

```diff
@@ -142,36 +146,49 @@ def file_leng(ubifs, inode):
         return fl
     return 0
 
-
-def _process_reg_file(ubifs, inode, path):
+def _process_reg_file(ubifs, inode, path, inodes):
     try:
         buf = bytearray()
+        start_key = 0x00 | (UBIFS_DATA_KEY << UBIFS_S_KEY_BLOCK_BITS)
```

Comment:
I know it is like this in some other place, but the `0x00 |` can be deleted, as it adds nothing to the result.